enterprise/server/invocation_search_service/invocation_search_service.go

```diff
@@ -29,15 +32,21 @@ const (
 	pageSizeOffsetPrefix = "offset_"
 )
 
+var (
+	olapInvocationSearchEnabled = flag.Bool("app.olap_invocation_search_enabled", true, "If true, InvocationSearchService will query clickhouse for some queries.")
+)
+
 type InvocationSearchService struct {
 	env environment.Env
 	h   interfaces.DBHandle
+	oh  interfaces.OLAPDBHandle
```

Comment:
nit: I don't think either `oh` and `h` are clear enough names for struct fields that are used throughout a package. They'd be okay as local function variables, but I'd prefer if struct names were even a tad more meaningful, like `dbh` and `olapdbh` for example.